ray-operator/controllers/ray/raycluster_controller_test.go

```diff
@@ -320,6 +320,58 @@ var _ = Context("Inside the default namespace", func() {
 				time.Second*(utils.RAYCLUSTER_DEFAULT_REQUEUE_SECONDS+5), time.Millisecond*500).Should(Equal(rayv1.Suspended))
 		})
 
+		It("set suspend to false and then revert it to true before all Pods are running", func() {
+			// suspend a Raycluster and check that all pods are deleted.
```

Comment:
Where do we check that all pods are deleted?

Reply:
Oh, it looks like in this part we actually set suspend to false, and we don't expect pods to be deleted.  So maybe this code comment just needs to be updated

Reply:
```suggestion
			// set suspend to false
```